src/main/twirl/views/sectionPreview.scala.html

```diff
@@ -13,7 +13,7 @@
     f"RIFS$id%05d"
 }
 
-@main(s"Event title", backLink=Some(BackLink("Return to application overview",controllers.routes.ApplicationController.show(appOverview.id).url))) {
+@main(formSection.title, backLink=Some(BackLink("Return to application overview",controllers.routes.ApplicationController.show(appOverview.id).url))) {
```

Comment:
We need a pattern for all titles, so perhaps it's good to discuss it here.  Most gov.uk pages seem to follow the patten "$heading - GOV.UK" so perhaps we could adopt "$heading - RIFS" ?